src/items/itemsLandingPage.component.tsx

```diff
@@ -79,6 +79,8 @@ function ItemsLandingPage() {
 
   const { data: systemData } = useGetSystem(itemData?.system_id);
 
+  const { data: systemTypesData } = useGetSystemTypes();
```

Comment:
It might be worth making a good first issue for using https://github.com/ral-facilities/inventory-management-system-api/issues/543 once added?

Reply:
#1493